_data/team.yml

```diff
@@ -41,6 +41,17 @@
     FinTech industries, for companies spanning across North America and Africa. Toni is also an avid contributor and maintainer of
     <a href="https://github.com/tonerdo">open source projects</a> with a huge focus on language runtimes and developer tools.
 
+- name: Eugene Kolnick
+  url: "https://www.linkedin.com/in/eugenekolnick/"
+  title: Principal Software Engineer
+  image: /assets/img/headshots/headshot-eugene-kolnick.jpg
+  bio: |
+    Eugene has almost 12 years of Software Engineering under his belt. 11 of those years were spent in FinTech working on 
```

Comment:
s/Software Engineering/Software Engineering Experience

Reply:
Thanks Toni! I've fixed this.